libparsec/crates/client/tests/unit/workspace_history/stat_entry.rs

```diff
@@ -65,6 +67,7 @@ async fn ok_file(
     let wksp1_foo_v2_children_available_timestamp: DateTime = *env
         .template
         .get_stuff("wksp1_foo_v2_children_available_timestamp");
+    let mallory = env.local_device("mallory@dev1");
```

Comment:
Trivia: you can also do `let mallory_dev1_id = "mallory@dev1".parse::<DeviceID>().unwrap()` to directly obtain the DeviceID

(though don't feel obliged to modify this code ^^)